internal/pkg/release/version.go

```diff
@@ -20,6 +20,9 @@ const (
 // snapshot is a flag marking build as a snapshot.
 var snapshot = ""
 
+// fips is a flag for marking a FIPS compliant build.
+var fips = "false"
```

Comment:
Where is this ever set to a different value? This whole version logic seems overly complex.

Reply:
Yes, a FIPS build will replace the symbol with "true"